inference-engine/thirdparty/clDNN/src/program.cpp

```diff
@@ -1129,7 +1130,9 @@ void program_impl::set_layout_optimizer_attributes(layout_optimizer& lo) {
             (prim.type() != cldnn::mvn::type_id()
              || (prim.as<mvn>().input().get_output_layout().data_type != data_types::u8 &&
                  prim.as<mvn>().input().get_output_layout().data_type != data_types::i8)
-             || prim.as<mvn>().get_primitive()->across_channels))
+             || prim.as<mvn>().get_primitive()->across_channels) &&
+            prim.type() != cldnn::arg_max_min::type_id() &&
+            prim.type() != cldnn::mutable_data::type_id())
```

Comment:
Does this affect any other topologies?

Reply:
Yeah, it also slightly improves the performance on some icnet versions (~7% for fp32 and 30% for fp16). All other models seems to be unaffected.